src/install/kafka/whatsNext.mdx

```diff
@@ -0,0 +1,4224 @@
+        title="Configuration settings"
+    >
+## Configure KafkaBrokerSample and KafkaTopicSample collection [#broker-collection]
+
+The Kafka integration collects both Metrics and Inventory information. Check the **Applies To** column below to see the settings available to each collection:
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "150px" }}>
+        Setting
+      </th>
+
+      <th>
+        Description
+      </th>
+
+      <th>
+        Default
+      </th>
+
+      <th>
+        Applies To
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        **CLUSTER_NAME**
+      </td>
+
+      <td>
+        User-defined name to uniquely identify the cluster being monitored. **Required**.
+      </td>
+
+      <td>
+        N/A
+      </td>
+
+      <td style={{ "text-align": "center" }}>
+        M/I
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        **KAFKA_VERSION**
+      </td>
+
+      <td>
+        The version of the Kafka broker you're connecting to, used for setting optimum API versions. It must match -or be lower than- the version from the broker.
+
+        Versions older than 1.0.0 may be missing some features.
+
+        **Note that if the broker binary name is kafka_2.12-2.7.0 the Kafka api version to be used is 2.7.0, the preceding 2.12 is the Scala language version**.
+      </td>
+
+      <td>
+        1.0.0
+      </td>
+
+      <td style={{ "text-align": "center" }}>
+        M/I
+      </td>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        **AUTODISCOVER_STRATEGY**
+      </td>
+
+      <td>
+        the method of discovering brokers. Options are `zookeeper` or `bootstrap`.
+      </td>
+
+      <td>
+        zookeeper
+      </td>
+
+      <td style={{ 'text-align': 'center' }}>
+        M/I
+      </td>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        **METRICS**
+      </td>
+
+      <td>
+        Set to `true` to enable Metrics only collection.
+      </td>
+
+      <td>
+        false
+      </td>
+
+      <td style={{ 'text-align': 'center' }}/>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        **INVENTORY**
+      </td>
+
+      <td>
+        Set to `true` to enable Inventory only collection.
+      </td>
+
+      <td>
+        false
+      </td>
+
+      <td style={{ 'text-align': 'center' }}/>
+    </tr>
+  </tbody>
+</table>
+
+## Zookeeper autodiscovery arguments [#zookeeper-discovery]
+
+These are only relevant when the `autodiscover_strategy` option is set to `zookeeper`.
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "150px" }}>
+        Setting
+      </th>
+
+      <th>
+        Description
+      </th>
+
+      <th>
+        Default
+      </th>
+
+      <th>
+        Applies To
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        **ZOOKEEPER_HOSTS**
+      </td>
+
+      <td>
+        The list of Apache ZooKeeper hosts (in JSON format) that need to be connected.
```

Comment:
instead of "that need to be connected", could you work something like "that you need to connect"?